### PR TITLE
fix: removing extra divs, fixing styling on 2 tabs, everything in quill

### PR DESCRIPTION
**Pull request by agrim-deriv**

### CODE REVIEW

File: src/components/CustomCheckbox/index.tsx
Comment (ahmed-deriv) [inline src/components/CustomCheckbox/index.tsx:1]: Is it only UI changes or logics changes as well ?

File: src/features/dashboard/components/ApiTokenCard/index.tsx
Comment (ahmed-deriv) [inline src/features/dashboard/components/ApiTokenCard/index.tsx:1]: Same as here.

File: src/features/dashboard/update-app/AppUpdateForm/app-update-form.scss [hunk 1]
```diff
@@ -29,26 +29,54 @@
...
     align-items: center;
     justify-content: center;
-
-    button {
-      margin-inline: 8px;
-    }
+    gap: rem(0.8);
   }
 
   .mblk {
```
Comment (ahmed-deriv) [inline src/features/dashboard/update-app/AppUpdateForm/app-update-form.scss:35]: can you create root values and also create this style into global file to reuse this spacing on other pages/components if needed. Also use `rem` to all places as some have value in pixels and some have `rem`

**Review by ahmed-deriv — CHANGES_REQUESTED** [review]
There are few code files that belongs to different PR once its approved and merge. Will review again.

File: src/components/CustomCheckbox/index.tsx
Comment (agrim-deriv) [reply src/components/CustomCheckbox/index.tsx:1]: Mostly UI

**Review by prince-deriv — APPROVED** [review]

**Pull request merged**